packages/playground/website/playwright/e2e/deployment.spec.ts

```diff
@@ -112,15 +120,14 @@ test('offline mode – the app should load even when the server goes offline', a
 
 	await page.goto(`${url}`);
 	await website.waitForNestedIframes();
-	// @TODO a better check – screenshot comparisons will be annoying to maintain
-	await expect(page).toHaveScreenshot('website-online.png', {
-		maxDiffPixels,
-	});
+
+	await expect(website.page.getByLabel('Open Site Manager')).toBeVisible();
+	expect(wordpress.locator('body')).toContainText('Edit site');
 
 	server!.kill();
 	await page.reload();
 	await website.waitForNestedIframes();
-	await expect(page).toHaveScreenshot('website-online.png', {
-		maxDiffPixels,
-	});
+
+	await expect(website.page.getByLabel('Open Site Manager')).toBeVisible();
```

Comment:
Why move away from screenshots? I can think of reasons (e.g., test fragility) but am curious about yours.